srcpkgs/strace/template

```diff
@@ -1,19 +1,26 @@
 # Template file for 'strace'
 pkgname=strace
-version=5.9
+version=5.10
 revision=1
 build_style=gnu-configure
-configure_args="--with-libunwind"
+configure_args="--enable-silent-rules --with-libunwind"
 makedepends="libunwind-devel"
 short_desc="System Call Tracer"
 maintainer="Anthony Iliopoulos <ailiop@altatus.com>"
 license="LGPL-2.1-or-later"
 homepage="https://strace.io/"
+changelog="https://raw.githubusercontent.com/strace/strace/master/NEWS"
 distfiles="https://github.com/strace/strace/releases/download/v${version}/strace-${version}.tar.xz"
-checksum=39473eb8465546c3e940fb663cb381eba5613160c7302794699d194a4d5d66d9
+checksum=fe3982ea4cd9aeb3b4ba35f6279f0b577a37175d3282be24b9a5537b56b8f01c
 
 case "$XBPS_TARGET_MACHINE" in
 	aarch64*|ppc64*) configure_args+=" --enable-mpers=no" ;;
 	# work around https://github.com/strace/strace/issues/133
 	x86_64-musl) configure_args+=" --enable-mpers=no" ;;
 esac
```

Comment:
Given https://github.com/strace/strace/commit/1b57b2241aef3e3ee5ea4cd60772252b15254de8 , can you try to remove this block?

Reply:
I tried but configure fails on both platforms with
```
checking whether to enable m32 personality support... no
configure: error: Cannot enable m32 personality support
```

Reply:
Ah, shame :/